src/main/kotlin/lotto/controller/PurchasesController.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.controller
+
+
+import lotto.dto.PurchaseResult
+import lotto.dto.UserPurchase
+
+object PurchasesController {
+    fun getPurchaseResult(): PurchaseResult {
+        val userPurchase = getUserPurchases()
+        return PurchaseResult(userPurchase)
+    }
+
+    private fun getUserPurchases(): UserPurchase {
+        val amount = Prompter.askAmount()
+        val manualTicketsCount = Prompter.askManualTicketsNumbers(amount)
```

Comment:
I understood these controllers are separated!
They're get all DTO from user before in earnest domain logic, right?